doc/content/en/docs/++version++/Editor Support/_index.md

```diff
@@ -0,0 +1,175 @@
+
+<img style="float: right; object-fit: contain; width: 160px; height: 160px;" src="eclipseAvroclipse.png"/>[Extension](https://marketplace.eclipse.org/content/avroclipse) ([source](https://github.com/dvdkruk/avroclipse))
+
+Released in 2015, this plugin received its last update in December 2019.
+
+Features for IDL files:
+
+* Syntax Highlighting
+* Code Completion
+* Error Highlighting
+
+## Emacs
+
+Sometimes ridiculed as an operating system, Emacs is a rich text editor. Many packages are available
+via one of the three main package archives: [GNU ELPA](https://elpa.gnu.org/), [non-GNU ELPA](https://elpa.nongnu.org/)
+or [MELPA](https://melpa.org/).
+
+### emacs-avro
+
+[Package source](https://github.com/logc/emacs-avro)
+
+Originally released elsewhere in 2013, the last update was in March 2021.
+
+This package is not available via the main package archives.
+
+Provides syntax highlighting for IDL schemata.
+
+## JetBrains IDE family
+
+JetBrains features an entire line of IDEs. They include IntelliJ IDEA, PyCharm, PhpStorm, GoLand, Rider, CLion,
+RustRover, WebStorm, RubyMine,
+DataGrip, DataSpell, ReSharper, Fleet, and Aqua.
+
+All of them use the [Plugin Marketplace](https://plugins.jetbrains.com/) to load extensions from.
+
+### Apache Avro IDL Schema Support
+
+<img style="float: right; object-fit: contain; width: 160px; height: 160px;" src="jetbrainsAvroIdlSchemaSupport.svg"/>[Plugin](https://plugins.jetbrains.com/plugin/15728-apache-avro-idl-schema-support) ([source](https://github.com/opwvhk/avro-schema-support?tab=readme-ov-file#intellij-plugin-for-apache-avro-idl))
+
+Released in 2021, this plugin received its last update in November 2023.
+
+Features for Avro schema & protocol definitions:
+
+* Syntax Highlighting
+* Code Completion
+* Code Formatting
+* Error Highlighting
+* Inspections & quick fixes (IDL only)
+* New file templates
+
+The plugin supports all JetBrains products.
+
+### Avro and Parquet Viewer
+
+<img style="float: right; object-fit: contain; width: 160px; height: 160px;" src="jetbrainsAvroAndParquetViewer.svg"/>[Plugin](https://plugins.jetbrains.com/plugin/12281-avro-and-parquet-viewer) ([source](https://github.com/benwatson528/intellij-avro-parquet-plugin))]
+
+Released in 2021, this plugin received its last update in November 2022.
+
+Allows previewing `.avro` files, and shows its schema in an editor tab.
+
+### Big Data File Viewer
+
+<img style="float: right; object-fit: contain; width: 160px; height: 160px;" src="jetbrainsBigDataFileViewer.svg"/>[Plugin](https://plugins.jetbrains.com/plugin/21701-big-data-file-viewer) (
+part of the [Big Data Tools](https://plugins.jetbrains.com/bundles/8-big-data-tools))
+
+Released in 2023, this plugin is actively developed by JetBrains.
+
+Allows previewing `.avro` files, and shows its schema in the structure tool window.
+
+## Pulsar/Atom
+
+Though the Atom.io editor has been sunset late 2022, the public fork [Pulsar](https://pulsar-edit.dev/) has taken its
```

Comment:
1. My earlier comment about Atom could be ignored!
2. Or we could drop it here too and mention just Pulsar.

Reply:
Thank you!
I'll go for option 2; that gives users more options.